examples/react-urql-retry/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
Just as the above says, extraneous files as we won't need tests added for the example. I assume they're from the boilerplate

Reply:
yes, removed!